src/express.js

```diff
@@ -854,7 +837,22 @@ export function generateClientLibrary() {
   }
 }
 
+function compile(code) {
+  const result = ts.transpileModule(code, {
```

Comment:
Not sure if this function gets called during runtime, but if so, may need to move `typescript` to a full dep instead of dev dep.